cmd/crictl/sandbox.go

```diff
@@ -172,12 +166,12 @@ var listPodSandboxCommand = cli.Command{
 			Usage: "filter by pod sandbox namespace",
```

Comment:
Maybe we can add shortname for namespace?

Reply:
`last` is using the shortname `n`.